internal/server/ftp/client.go

```diff
@@ -78,7 +77,8 @@ func (c *Client) Common() model.Common {
 
 // ReadDir fetches the contents of a directory, returning a list of os.FileInfo's
 func (c *Client) ReadDir(path string) ([]os.FileInfo, error) {
-	return c.ftp.ReadDir(regexp.QuoteMeta(path))
+	//return c.ftp.ReadDir(regexp.QuoteMeta(path))
+	return c.ftp.ReadDir(path)
```

Comment:
Please create an other PR about this one.

Reply:
This pushed after RP, sorry, I create new branch for it.